content/en/docs/setup/best-practices/certificates.md

```diff
@@ -66,15 +66,17 @@ Required certificates:
 
 | Default CN                    | Parent CA                 | O (in Subject) | kind                                   | hosts (SAN)                                 |
 |-------------------------------|---------------------------|----------------|----------------------------------------|---------------------------------------------|
-| kube-etcd                     | etcd-ca                   |                | server, client                         | `localhost`, `127.0.0.1`                        |
+| kube-etcd                     | etcd-ca                   |                | server, client                         | `localhost`, `127.0.0.1`, (See annotation [1])           |
 | kube-etcd-peer                | etcd-ca                   |                | server, client                         | `<hostname>`, `<Host_IP>`, `localhost`, `127.0.0.1` |
 | kube-etcd-healthcheck-client  | etcd-ca                   |                | client                                 |                                             |
 | kube-apiserver-etcd-client    | etcd-ca                   | system:masters | client                                 |                                             |
-| kube-apiserver                | kubernetes-ca             |                | server                                 | `<hostname>`, `<Host_IP>`, `<advertise_IP>`, `[1]` |
+| kube-apiserver                | kubernetes-ca             |                | server                                 | `<hostname>`, `<Host_IP>`, `<advertise_IP>`, (See annotation [2]) |
```

Comment:
@pierreilki mixing notes and data can be confusing (and sometimes impedes accessibility). With more remarks the issue becomes more acute. Is there another way to add remarks? Maybe adding an extra column? Could a &dagger; symbol work instead?

Reply:
@sftim Both annotations are here to explain what are the others SAN entries that have to be set in the CSR. So i don't think that adding a new column will fit the need. And i don't know if  a † symbol will solve the problem because annotations [1] and [2] describe 2 different cases, with 2 different SAN entries...

Reply:
No worries. Let's merge this change and then track another issue about tidying up the page, there are a few tweaks to make.